### ISSUE TITLE
How to get mobilenetV2 normalization values?

### ISSUE THREAD

**Issue by Regenhardt** [tensorflow/tensorflow#57817]
### 1. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- TensorFlow installation (pip package or built from source): pip
- TensorFlow library (version, if pip package or github SHA, if built from source): 2.9.2

### 2. Code

Provide code to help us reproduce your issues using one of the following options:

Same problem as #47559:

Input tensor has type kTfLiteFloat32: it requires specifying NormalizationOptions metadata to preprocess input images.

So I added metadata:  
```
from tflite_support.metadata_writers import object_detector
from tflite_support.metadata_writers import writer_utils

# Create metadata writer
writer = object_detector.MetadataWriter.create_for_inference(
    writer_utils.load_file(tflite_file),
    [127.5],
    [127.5],
    label_file_paths=[label_file])

print("Identified metadata: ", writer.get_metadata_json())

# Write metadata
writer_utils.save_file(writer.populate(), tflite_file)
```

### 3. Failure after conversion

When running, it still complains about not having normalization values. Tried both [127.5],[127.5] and [255],[0] as values, but neither work. How can i find out what the models expects as normalization values?  
I loaded the saved model in the netron.app and the input just says `type: float32[1,640,640,3]`

**Comment by mohantym** [comment]
Hi @Regenhardt !
Could you check with input_norm_mean = 0 ,input_norm_std = 255 and [ssd_mobilenet_v2_fpnlite_640x640_coco17_tpu-8/model.tflite](https://tfhub.dev/tensorflow/ssd_mobilenet_v2/fpnlite_640x640/1)
```

LABEL_PATH = 'label_map.txt'
MODEL_PATH = "ssd_mobilenet_v2_fpnlite_640x640_coco17_tpu-8/model.tflite"
SAVE_TO_PATH = "ssd_mobilenet_v2_fpnlite_640x640_coco17_tpu-8/model_with_metadata.tflite"

# Step 1: Specify the preprocessing parameters and label file
writer = object_detector.MetadataWriter.create_for_inference(
    writer_utils.load_file(MODEL_PATH), input_norm_mean=[0],
    input_norm_std=[255], label_file_paths=[LABEL_PATH])

# Step 2: Export the model with metadata
writer_utils.save_file(writer.populate(), SAVE_TO_PATH)
```
[Reference.](https://blog.tensorflow.org/2021/06/easier-object-detection-on-mobile-with-tf-lite.html)

Thank you!

**Comment by Regenhardt** [comment]
It works! Tried with my model, and it also works. My guess is I copied or wrote the wrong file somewhere between exporting the model and adding it to the android project.

So now, again, I tried with both [127.5],[127.5] and [0],[255] - both work the same, as far as I can tell. Any idea why? Shouldn't one of them crash or something? 

How do I know what values my specific model needs?

**Comment by mohantym** [comment]
@Regenhardt !
Thanks for the udpate.

@sachinprasadhs !
Could you look at this issue.

Thank you!

**Comment by pjpratik** [comment]
Hi @Regenhardt 

The normalization which the TFLite model expects depends upon the model from which it is being converted.

The Mobilenetv2 from keras application applies the tf normalization which normalizes to the output range [-1,1]. So we may have provide [127.5],[127.5] values in metadata. 

https://github.com/keras-team/keras-applications/blob/06fbeb0f16e1304f239b2296578d1c50b15a983a/keras_applications/mobilenet.py#L84

https://github.com/keras-team/keras-applications/blob/06fbeb0f16e1304f239b2296578d1c50b15a983a/keras_applications/imagenet_utils.py#L42

Where as the input images for the tensorflow hub [MobileNetV2](https://tfhub.dev/google/imagenet/mobilenet_v2_100_224/classification/5) model are expected to have color values in the range [0,1] (provide [0],[255] values in metadata). 

Thanks.
